test/example/test.ts

```diff
@@ -5,93 +5,57 @@
 /// <reference types="@types/node" />
 import test from 'ava';
 import cbor from 'cbor';
-import {
-  UDPDevice,
-  MockNetwork,
-  RemoteAddressInfo,
-} from '../../src/platform/mock-radio';
-import {
-  MockLocalHomePlatform,
-  UDPScanConfig,
-  ScanState,
-} from '../../src/platform/mock-local-home-platform';
+import {UDPDevice, MockNetwork} from '../../src/platform/mock-radio';
 import {injectSmarthomeStubs} from '../../src/platform/stub-setup';
+import {MockLocalHomePlatform} from '../../src/platform/mock-local-home-platform';
+
+// Common address for all UDP messaging
+const updAddress: string = '255.255.255.255';
 
 // Tests a UDP identify flow end-to-end
 test('udp-device-connects', async t => {
-  // First, create a scan configuration
-  const scanConfig = new UDPScanConfig(
-    ScanState.Unprovisioned,
-    '255.255.255.255',
-    3311,
-    3312,
-    'A5A5A5A5'
-  );
-
   // Mock a UDP Network
   const mockNetwork = new MockNetwork();
 
   // Mock the Local Home Platform
   const mockLocalHomePlatform = MockLocalHomePlatform.getInstance();
 
-  mockLocalHomePlatform.initializeRadio(mockNetwork, [scanConfig]);
-
-  // Mock a UDP Device
-  const mockDevice = new UDPDevice();
-
-  const deviceId = 'test-device-id';
-  // Device data that mock device sends back
-
-  const discoveryPort = 12345;
-  const discoveryData = {
-    id: deviceId,
-    model: '2',
-    hw_rev: '0.0.1',
-    fw_rev: '1.2.3',
-    channels: [discoveryPort],
-  };
-
-  // Sample device response
-  mockDevice.setUDPMessageAction((msg: Buffer, rinfo: RemoteAddressInfo) => {
-    const packetBuffer = Buffer.from(scanConfig.discoveryPacket, 'hex');
-    if (msg.compare(packetBuffer) !== 0) {
-      console.warn('UDP received unknown payload:', msg, 'from:', rinfo);
-      return;
-    }
-    console.debug('UDP received discovery payload:', msg, 'from:', rinfo);
-
-    const discoveryBuffer = cbor.encode(discoveryData);
-
-    // TODO(cjdaly) Add error path
-    mockNetwork.sendUDPMessage(
-      discoveryBuffer,
-      rinfo.port,
-      rinfo.address,
-      discoveryPort,
-      scanConfig.broadcastAddress
-    );
-  });
-
-  //Mock network needs to listen for device response
-  mockNetwork.registerUDPListener(
-    mockDevice,
-    scanConfig.broadcastPort,
-    scanConfig.broadcastAddress
+  const mockDevice = new UDPDevice(
```

Comment:
is the mockDevice still needed? you could simply pass the deviceId and devicePort as parameter for trigger identify.

Reply:
Removed `mock-radio.ts` for this PR since nothing is being used yet.  I anticipate adding it back for #23 and #16